services/simcore/docker-compose.yml

```diff
@@ -433,6 +524,13 @@ services:
       placement:
         constraints:
           - node.labels.rabbit==true
+      resources:
+        limits:
+          memory: 1G
+          cpus: '1.000'
+        reservations:
+          memory: 256M
+          cpus: '0.1'
```

Comment:
this should probably also be well monitored. where do you have the references for such limits?

Reply:
As discussed in whereby